src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/DataFormats.cs

```diff
@@ -1,472 +1,190 @@
-
-        /// <summary>
-        /// Specifies the standard Windows original equipment
-        /// manufacturer (OEM) text format. This field is read-only.
-        /// </summary>
-        public static readonly string OemText = "OEMText";
-
-        /// <summary>
-        /// Specifies the Windows palette format. This
-        /// field is read-only.
-        /// </summary>
-        public static readonly string Palette = "Palette";
-
-        /// <summary>
-        /// Specifies the Windows pen data format, which consists of
-        /// pen strokes for handwriting software; This field is read-only.
-        /// </summary>
-        public static readonly string PenData = "PenData";
-
-        /// <summary>
-        /// Specifies the Resource Interchange File Format (RIFF)
-        /// audio format. This field is read-only.
-        /// </summary>
-        public static readonly string Riff = "RiffAudio";
-
-        /// <summary>
-        /// Specifies the wave audio format, which Windows Design does not
-        /// directly use. This field is read-only.
-        /// </summary>
-        public static readonly string WaveAudio = "WaveAudio";
-
-        /// <summary>
-        /// Specifies the Windows file drop format, which Windows Design
-        /// does not directly use. This field is read-only.
-        /// </summary>
-        public static readonly string FileDrop = "FileDrop";
-
-        /// <summary>
-        /// Specifies the Windows culture format, which Windows Design does
-        /// not directly use. This field is read-only.
-        /// </summary>
-        public static readonly string Locale = "Locale";
-
-        /// <summary>
-        /// Specifies text consisting of HTML data. This
-        /// field is read-only.
-        /// </summary>
-        public static readonly string Html = "HTML Format";
-
-        /// <summary>
-        /// Specifies text consisting of Rich Text Format (RTF) data. This
-        /// field is read-only.
-        /// </summary>
-        public static readonly string Rtf = "Rich Text Format";
-
-        /// <summary>
-        /// Specifies a comma-separated value (CSV) format, which is a
-        /// common interchange format used by spreadsheets. This format is not used directly
-        /// by Windows Design. This field is read-only.
-        /// </summary>
-        public static readonly string CommaSeparatedValue = "CSV";
-
-        /// <summary>
-        /// Specifies the Windows Design string class format, which Win
-        /// Forms uses to store string objects. This
-        /// field is read-only.
-        /// </summary>
-        public static readonly string StringFormat = typeof(string).FullName;
-
-        /// <summary>
-        /// Specifies a format that encapsulates any type of Windows Design
-        /// object. This field is read-only.
-        /// </summary>
-        public static readonly string Serializable = "PersistentObject";
```

Comment:
`DataFormatNames.Serializable` seems to be `WindowsForms10PersistentObject` rather than `PersistentObject`. Isn't that a breaking change?